Kickstarter-iOS/Features/PledgeOverTime/Views/PledgePaymentPlanOptionView.swift

```diff
@@ -76,6 +84,8 @@ final class PledgePaymentPlanOptionView: UIView {
       ),
       for: .normal
     )
+
+    self.ineligibleBadgeLabel.text = "Available for pledges over $150"
```

Comment:
Two thoughts:

1. The $150 minimum - or even better, the whole badge text! - should come from the API. It seems likely to change.
2. This needs a `//todo` comment for translations

Reply:
Raising this in `#ppo-mobile-squad`